ci/requirements/doc.yml

```
@@ -17,6 +15,12 @@ dependencies:
   - sphinx-autosummary-accessors
   - sphinx-copybutton
   - xarray
+  # For examples
+  - adlfs
+  - nbsphinx
+  - ipykernel
```

Alphabetical nitpick.
```suggestion
  - ipykernel
  - nbsphinx
```